api/scm.go

```diff
@@ -164,14 +169,50 @@ func SyncRepo(c *gin.Context) {
 		// set repo to inactive - do not delete
 		r.SetActive(false)
 		// update repo in database
-		e := database.FromContext(c).UpdateRepo(r)
-		if e != nil {
+		err := database.FromContext(c).UpdateRepo(r)
+		if err != nil {
 			retErr := fmt.Errorf("unable to update repo for org %s: %w", org, err)
 
 			util.HandleError(c, http.StatusInternalServerError, retErr)
 
 			return
 		}
+		err = updateUserFavorites(c, r)
```

Comment:
I think we're missing a space here:

```suggestion
		}

		err = updateUserFavorites(c, r)
```